src/main/webapp/dashboard.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+<title>Dashboard</title>
+
+<!-- Bootstrap CSS only -->
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+
+<!-- Custom styles -->
+<link rel="stylesheet" href="style.css">
+
+<div class="container mt-3">
+  <br>
+  <!-- Dashboard Nav tabs -->
+  <ul class="nav nav-tabs">
+    <li class="nav-item">
+      <a class="nav-link active" data-toggle="tab" href="#abt-me">About Me</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" data-toggle="tab" href="#affiliations">Affiliations</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" data-toggle="tab" href="#int-mentees">Interested Mentees</a>
+    </li>
+  </ul>
+
+  <!-- Tab panes -->
+  <div class="tab-content">
+    <!-- About Me -->
+    <div id="abt-me" class="container tab-pane active"><br>
```

Comment:
Should probably avoid abbreviations where possible to preserve readability. Could change to `about-me`.